src/EmojiInput.js

```diff
@@ -287,6 +287,9 @@ class EmojiInput extends React.PureComponent {
             return e1.char !== e2.char;
         });
 
+        this.filtered = this.props.filterFunctions.length === 0
```

Comment:
Let's directly connect the filter functions to lodash chain, e.g.
```js
_(emojis)
  .filter(emoji => _.every ...)
  ...
```

Also `_.every` will return true if the first argument is empty array, so it's safe not to check whether the length is zero

Reply:
Done. :+1: